rascal-textmate-core/src/main/rascal/lang/rascal/grammar/analyze/Delimiters.rsc

```diff
@@ -193,9 +193,9 @@ default Maybe[Symbol] unique(set[Maybe[Symbol]] _) = nothing();
 }
 
 bool isDelimiter(lit(string))
-    = /^\w+$/ !:= string;
+    = /^\W+$/ := string;
 bool isDelimiter(cilit(string))
```

Comment:
how about we redirect `lit` to `cilit`? so `bool isDelimiter(lit(string)) = isDelimiter(cilit(string));` so document we do the same?